packages/arb-token-bridge-ui/src/hooks/useDisabledFeatures.ts

```diff
@@ -2,10 +2,14 @@ import { useCallback } from 'react'
 import { useArbQueryParams, DisabledFeatures } from './useArbQueryParams'
 
 export const useDisabledFeatures = () => {
-  const [{ disabledFeatures }] = useArbQueryParams()
+  const [{ disabledFeatures, mode }] = useArbQueryParams()
 
   const isFeatureDisabled = useCallback(
     (feature: DisabledFeatures) => {
+      if (feature === DisabledFeatures.TRANSFERS_TO_NON_ARBITRUM_CHAINS) {
```

Comment:
Why do we need to make it widget specific? I get it that it will be used with the widget, but shouldn't we make the architecture more agnostic?